qiskit/circuit/library/__init__.py

```diff
@@ -113,6 +113,7 @@
    SdgGate
    SwapGate
    iSwapGate
+   SiSwapGate
```

Comment:
The SiSwap Gate should also be added to the documentation in this init file, and removed from the docs in circuit/library/standard_gates. 
See the following PR: #9668